### ISSUE TITLE
`controller.signal()`: return signal instance

### ISSUE THREAD

**Issue by Guria** [cerebral/cerebral#145]
Sometimes very useful to get signal instance upon creation.
Also would be great to have signalName exposed on it.
Example:
```js
// before
const signal = function (name, chain) {
  controller.signal(name, chain);
  return name;
}

const routes = {
  '/': signal('home.opened', chains.home.opened),
  '*': signal('notFound.opened', chains.notFound.opened)
};

Router(controller, routes);

// after

const routes = {
  '/': controller.signal('home.opened', chains.home.opened).name,
  '*': controller.signal('notFound.opened', chains.notFound.opened).name
};

Router(controller, routes);

```

**Comment by Guria** [comment]
cc @garth

**Comment by garth** [comment]
Looks like a great idea when passing around chains instead of signals for route creation.

**Comment by christianalfoni** [comment]
Okay, now returning signal on creating and added .signalName property as well. Can not use .name as it is a function we are returning

**Comment by Guria** [comment]
fixed in 0.27
